app/models/concerns/sufia/solr_document_behavior.rb

```diff
@@ -57,6 +57,16 @@ def collection_ids
       Array.wrap(self['collection_ids_tesim'])
     end
 
+    def embargo_release_date
+      date_field_name = Hydra.config.permissions.embargo.release_date.sub(/_dtsi/, '')
```

Comment:
@mjgiarlo I think ya'll know a lot more about this than I do, so sounds good to me. Agree with @awead that it is more straight-forward.